python/mlc_chat/compiler/flags_model_config_override.py

```diff
@@ -30,3 +32,29 @@ def apply(self, model_config):
             model_config.max_batch_size = self.max_batch_size
         if self.num_shards is not None:
             model_config.num_shards = self.num_shards
+
+        # Handle sliding window and sliding window chunk size
```

Comment:
Is it even necessary to provide options to override `sliding_window` at all? For mistral models, I suppose it always comes with the model